corehq/apps/export/templates/export/partial/new_customize_export_templates.html

```diff
@@ -142,17 +150,30 @@ <h4>{% trans "Choose the fields you want to export." %}</h4>
                 </tfoot>
                 {% endif %}
 
+                {% if request|toggle_enabled:"EXPORT_MULTISORT" %}
+                <tbody data-bind="
+                    multi_sortable: {
+                        data: table.columns,
+                        as: 'column',
+                        name: 'ko-export-column-template'
+                    }
+                ">
+                {% else %}
                 <tbody data-bind="
                     sortable: {
                         data: table.columns,
                         as: 'column',
                         name: 'ko-export-column-template'
                     }
                 ">
+                {% endif %}
                     <tr data-bind="
                         visible: column.isVisible($parent),
-                        attr: {'data-order': _sortableOrder}
-                        ">
+                        attr: {'data-order': _sortableOrder},
+                        css: { isSelectedForExport: column.selected }
+                        "
+                        {% if request|toggle_enabled:"EXPORT_MULTISORT" %}style="cursor:move"{% endif %}
```

Comment:
@orangejenny What do you think about adding this styling to the row to suggest that the rows can be dragged?

Reply:
If you put the vertical arrows drag icon back and have a move cursor on that, I think that's enough to indicate draggability. I'd consider changing this style to a hand cursor, so the rows look clickable, since the selecting behavior is hard to discover (normally I'd suggest adding a checkbox, but since these rows already have checkboxes for a different purpose that doesn't work).